jinja2/utils.py

```diff
@@ -136,9 +136,7 @@ def import_string(import_name, silent=False):
         if ':' in import_name:
             module, obj = import_name.split(':', 1)
         elif '.' in import_name:
-            items = import_name.split('.')
-            module = '.'.join(items[:-1])
-            obj = items[-1]
+            module, _, obj = import_name.rpartition('.')
```

Comment:
`module, obj = import_name.rsplit('.', 1)`

Reply:
I used rpartition because it appears to be slightly faster than rsplit. 
But rsplit looks cleaner.

Reply:
@davidism 
Would you like me to change it to rsplit?